### ISSUE TITLE
SRW - allow visualization of all parameters from a Trajectory .dat file

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#862]
Need to allow all the columns to be available for visualization:
```
#ct [m], X [m], BetaX [rad], Y [m], BetaY [rad], Z [m], BetaZ [rad], Bx [T], By [T], Bz [T]
0.0	0.0	0.0	0.0	0.0	0.0	0.9999999854930839	0.0	0.0	0.0
0.0	0.0	0.0	0.0	0.0	0.00025552554884836773	0.9999999854930839	0.0	0.0	0.0
0.0	0.0	0.0	0.0	0.0	0.0005110510976967355	0.9999999854930839	0.0	0.0	0.0
0.0	0.0	0.0	0.0	0.0	0.0007665766465451032	0.9999999854930839	0.0	0.0	0.0
0.0	0.0	0.0	0.0	0.0	0.001022102195393471	0.9999999854930839	0.0	0.0	0.0
0.0	0.0	0.0	0.0	0.0	0.0012776277442418387	0.9999999854930839	0.0	0.0	0.0
0.0	0.0	0.0	0.0	0.0	0.0015331532930902064	0.9999999854930839	0.0	0.0	0.0
0.0	0.0	0.0	0.0	0.0	0.0017886788419385741	0.9999999854930839	0.0	0.0	0.0
0.0	0.0	0.0	0.0	0.0	0.002044204390786942	0.9999999854930839	0.0	0.0	0.0
......................
```
This will require some additional work in SRW source code.

**Comment by mrakitin** [comment]
Too many 1st priorities, reduce priority to 2nd. Will take care of it after other 1st priority issues are solved.

**Comment by mrakitin** [comment]
Implemented in bf1014112bacf7041a112fda69af49f22ebfb6f4.
